modules/face/src/precomp.hpp

```diff
@@ -48,11 +48,17 @@
 #include "opencv2/core/utility.hpp"
```

Comment:
Not sure if these changes are actually needed. This file can be left alone right?